From 241fd5cb24ddefd9d4e0d6272e3fed14d90712bf Mon Sep 17 00:00:00 2001
From: Oguzhan Yalcin <oyalcin@google.com>
Date: Thu, 28 Aug 2025 05:06:45 -0700
Subject: [PATCH 3/3] Move feature under the sysconfig folder to make it
 accessible by the device.

Bug: 436433817
Test: GtsOsTestCases
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:8b89163d61b800ca56ce974e3546790b8617da9d)
Merged-In: I244d30f466cd57e7757d23c986e7bd31cf84dca9
Change-Id: I244d30f466cd57e7757d23c986e7bd31cf84dca9
---
 patchlevel/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/patchlevel/Android.bp b/patchlevel/Android.bp
index 99d43233..ade7d7b3 100644
--- a/patchlevel/Android.bp
+++ b/patchlevel/Android.bp
@@ -17,5 +17,6 @@
 prebuilt_etc {
     name: "apexd.mainline_patch_level_2",
     src: "com.google.android.mainline.patchlevel.2.xml",
+    sub_dir: "sysconfig",
     filename_from_src: true,
 }
-- 
2.53.0

